### PR TITLE
Automated testing -> master

### PR DESCRIPTION
**Pull request by Salamandar**
There was a testing branch, i created a PR for you :)

### CODE REVIEW

File: manifest.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ description.fr = "Système de gestion de contenu basé sur Wiki"
 
 version = "26.2~ynh1"
 
-maintainers = ["eric_G"]
```
Comment (marclaporte) [inline manifest.toml:10]: I am a community member of Tiki Wiki CMS Groupware.

Is there anything I should know about? I can be reached in public or private.

Thanks!

**Comment by Salamandar** [comment]
!testme

File: manifest.toml (hunk 1 above)
Comment (Salamandar) [reply manifest.toml:10]: Just that @ericgaspar wants to reduce the amount of pressure and pings he can receive (not from tiki especially, but in general). We are not dropping the package, don't worry.

Comment (marclaporte) [reply manifest.toml:10]: I appreciate everything @ericgaspar did and I thank you @Salamandar for stepping up.

Comment (Salamandar) [reply manifest.toml:10]: I'm not stepping up to become this package's maintainer, but we are still trying to maintain it (at least update it / fix when bugs). :)

Comment (marclaporte) [reply manifest.toml:10]: @Salamandar: Understood. @rodriguezny can maintain it. He is a good coder, but new to YunoHost. Can you guide him to learn?

Thanks!

**Pull request merged**